model/src/main/java/com/basistech/rosette/dm/AnnotatedText.java

```diff
@@ -47,18 +48,19 @@
  * For Json, and Java, the 'adm-json' module provides the supported serialization.
  */
 @SuppressWarnings("deprecation")
-public class AnnotatedText {
+public class AnnotatedText implements Serializable {
+    private static final long serialVersionUID = 222L;
```

Comment:
I was thinking we'd just put the version number for the release of the ADM when we make a change. But if you think that just starting everything at one would be better, I'll do it.